parquet/transformer.go

```diff
@@ -177,6 +177,7 @@ func (ReverseTransformer) ReverseTransformValues(table *schema.Table, values []a
 			if err := t.Set(v); err != nil {
 				return nil, fmt.Errorf("failed to convert value %v to type %s: %w", v, table.Columns[i].Type, err)
 			}
+			res[i] = t
```

Comment:
Without this, the read-write test panics if there's a type error, instead of failing with a proper (and helpful) error message.